examples/spack/Dockerfile

```diff
@@ -59,24 +62,16 @@ RUN spack compiler find --scope system
 # Test: Some basic commands.
 RUN which spack
 RUN spack --version
+RUN spack compiler find
 RUN spack compiler list
 RUN spack compiler list --scope=system
 RUN spack compiler list --scope=user
 RUN spack compilers
-RUN spack spec netcdf
+RUN spack spec charliecloud
 
 # Test: Install Charliecloud.
-RUN spack spec charliecloud+builder
-RUN spack install charliecloud+builder
-
-# Test: Run Spack test suite. Distro package dependencies discovered by trial
-# and error.
-#
-# FIXME: Commented out because the suite fails. Last time I tried, 6 tests
-# failed and 1 "xfailed".
-#RUN apt-get install -y \
-#    python-pkg-resources
-#RUN spack test
```

Comment:
So this wasn't doing anything because it was commented out. Do we want to retain the documentation that the Spack test suite doesn't work?

Reply:
Couple of thoughts: 1) it has been a very long time (2+ years?) since I personally tried to run spack unit tests in a container; and 2) even if they did work, I'm not so sure there is much value in including them in this example. This image takes quite a long time to build, and working unit tests would make it worse.

I would argue that unless we think we have something to gain by including them in the example, we don't need to comment on them.